File: ClientApp/classes/signupvm.ts
export class SignUpVM {
    FirstName: string;
    NicName: string;
    Email: string;
    Password: string;


    constructor(firstName: string, NicName: string, email: string, password: string) {
        this.FirstName = firstName;
        this.NicName = NicName;
        this.Email = email;
        this.Password = password;
    }
}

File: Data/DTO/SignUpVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SignalRChat.Data.DTO
{
    public class SignUpVM
    {
        public string FirstName { get; set; }
        public string NicName { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
    }
}
